### PR TITLE
Security update for Qt

### PR DESCRIPTION
**Pull request by spham-amzn**
- Split build.sh to build-darwin.sh and build-linux.sh
- Updated argument call to Qt's configure to add '--tiff=system' to use a specific libtiff in place of the built-in libtiff source that comes with the qt 5.15.2 src
- Add 3P dependency to O3DE libtiff and zlib in order to use for the libtiff replacement (zlib is needed because overriding tiff to use the system assumes its linking to a shared library)
- Removed QtWebEngine/QtPdf from the build
- Removed Miniconda/python 2.7 dependency needed for QtWebEngine/QtPdf
- Updated build-linux.sh to include information and checks for required pacakges
- Renamed build.bat to build-windows.bat for consistency
- Update install_windows_extras.py to download to the temp folder
- Update package list to build the intermediate package to the temp folder instead
- Fix libqxcb-glx-integration.so to the correct subfolder on Linux
- Add code and files to copy over additional copyright notices
- Update mac build script to make sure xcode is either version 10 or 11

Signed-off-by: Steve Pham <82231385+spham-amzn@users.noreply.github.com>

### CODE REVIEW

**Review by Jackie9527 — APPROVED** [review]
LGTM

**Review by cgalvan — APPROVED** [review]
It might just be this new beta version of GH's files changed viewer, but did you delete the `install_miniconda.py` script?  It's not showing up as deleted, but we don't need it anymore now that you removed the webengine build, so it can be safely deleted.

**Comment by spham-amzn** [comment]
> It might just be this new beta version of GH's files changed viewer, but did you delete the `install_miniconda.py` script? It's not showing up as deleted, but we don't need it anymore now that you removed the webengine build, so it can be safely deleted.

Ah, it has not been deleted. I will remove that as well, good catch

File: package-system/Qt/install_windows_extras.py [hunk 1]
```diff
@@ -18,25 +18,37 @@
...
-    installer_link = dependencies[name]
-    file_name = os.path.basename(installer_link)
+try:
 
-    # Download the zip (if we haven't already, so that this script can be run iteratively)
-    if not os.path.exists(file_name):
-        print(f"Downloading {name} from {installer_link}")
+    # Set the current working directory to temp
+    current_dir = os.getcwd()
+    os.chdir("temp")
```
Comment (lmbr-pip) [inline package-system/Qt/install_windows_extras.py:27]: Nit: Any reason to do this over tempfile.mkdtemp() ?

File: package-system/Qt/install_windows_extras.py [hunk 2]
```diff
@@ -18,25 +18,37 @@
...
+        print(f"Attempting to install {name}")
 
-        print(f"Download of {name} complete => {os.path.abspath(file_name)}")
+        installer_link = dependencies[name]
+        file_name = os.path.basename(installer_link)
 
-    # We will unzip the package into the local temp directory
-    install_dir = os.path.abspath(os.path.join('temp', name))
-    print(f"Installing {name} to {install_dir}")
+        # Download the zip (if we haven't already, so that this script can be run iteratively)
```
Comment (lmbr-pip) [inline package-system/Qt/install_windows_extras.py:36]: nit: Avoid 'We' in scripts etc, not clear to future maintainers who 'We' is often (a person, a script, a system etc)

**Review by lmbr-pip — APPROVED** [review]
LG. Thanks for doing this.

File: package-system/Qt/install_windows_extras.py (hunk 1 above)
Comment (spham-amzn) [reply package-system/Qt/install_windows_extras.py:27]: The build script works off of a temp subdirectory and everything is done there. We could use a local temp dir with cleanup, but having a fixed temp folder makes developing the scripts much easier (for incremental reasons).

**Review by Jackie9527 — APPROVED** [review]
LGTM

**Review by lawsonamzn — APPROVED** [review]

**Review by Jackie9527 — APPROVED** [review]
LGTM

**Pull request merged**